predicthq/endpoints/v1/events/schemas.py

```diff
@@ -1,33 +1,32 @@
-from schematics.common import NONEMPTY
-
 from predicthq.endpoints.schemas import (
-    PaginatedMixin,
-    SortableMixin,
-    Model,
-    ResultSet,
-    ListType,
-    StringType,
-    GeoJSONPointType,
-    StringListType,
-    StringModelType,
     Area,
-    ModelType,
-    IntRange,
-    IntType,
+    BooleanType,
+    BrandUnsafe,
+    DateAround,
     DateTimeRange,
     DateTimeType,
-    FloatType,
-    ResultType,
-    DictType,
     DateType,
-    Place,
-    DateAround,
-    LocationAround,
-    BooleanType,
-    BrandUnsafe,
+    DictType,
     Entity,
+    FloatType,
+    GeoJSONPointType,
+    IntRange,
+    IntType,
+    ListType,
+    LocationAround,
+    Model,
+    ModelType,
+    PaginatedMixin,
+    Place,
     PolyModelType,
+    ResultSet,
+    ResultType,
+    SortableMixin,
+    StringListType,
+    StringModelType,
+    StringType,
```

Comment:
Sorry, this is just sorting imports by black. I should have done that as a separate commit.